src/Text/KirbyTags.php

```diff
@@ -22,7 +22,7 @@ class KirbyTags
 
     public static function parse(string $text = null, array $data = [], array $options = []): string
     {
-        return preg_replace_callback('!(?=[^\]])\([a-z0-9_-]+:.*?\)!is', function ($match) use ($data, $options) {
+        return preg_replace_callback('!(?=[^\]])(?=\([a-z0-9_-]+:)(\((?:[^()]+|(?1))*+\))!is', function ($match) use ($data, $options) {
```

Comment:
It would be nice to move the RegEx into a separate string and document it with comments like this:

```php
$regex = '!
            (?=[^\]])          # What this part does
            (?=\([a-z0-9_-]+:) # And this one
            ...
          !isx';